entries/a/app.flexera.com.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Software Vulnerability Research": {
```

Comment:
Should it be `Flexera Software Vulnerability Research` or `Flexera One`?

Reply:
@ApCoder123 suggested Software Vulnerability Research

Reply:
Whoops. I did mean to write `Flexera Software Vulnerability Research` in my original review. Sorry for not making that clear.

Reply:
```suggestion
  "Flexera Software Vulnerability Research": {
```